python/paddle/fluid/contrib/slim/tests/qat2_int8_image_classification_comparison.py

```diff
@@ -318,6 +330,8 @@ def test_graph_transformation(self):
         _logger.info('Batch number: {0}'.format(batch_num))
         _logger.info('Accuracy drop threshold: {0}.'.format(acc_diff_threshold))
         _logger.info('Quantized ops: {0}.'.format(self._quantized_ops))
```

Comment:
Hi @wojtuss  I have one not important point, When ops_to_quantize is empty, this log is empty set too. But I think we should show op list that has use_quantizer because if it is empty we quantize all quantizable ops right ?

Reply:
@lidanqing-intel , good point, this log is not as useful as it could be. I will try to improve it.